test/others/libcriu/test_pre_dump.c

```diff
@@ -0,0 +1,153 @@
+#include "criu.h"
+#include <fcntl.h>
+#include <stdio.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <signal.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <sys/wait.h>
+#include <unistd.h>
+#include "lib.h"
+
+static int wdir_fd, cur_imgdir = -1;
+
+static int stop = 0;
+static void sh(int sig)
+{
+	stop = 1;
+}
+
+static void open_imgdir(void)
+{
+	char p[10];
+	static int id = 0;
+
+	if (id > 0) {
+		sprintf(p, "../dir-%d", id);
+		criu_set_parent_images(p);
+	}
+	if (cur_imgdir != -1)
+		close(cur_imgdir);
+	sprintf(p, "dir-%d", ++id);
+	mkdirat(wdir_fd, p, 0700);
+	cur_imgdir = openat(wdir_fd, p, O_DIRECTORY);
+	criu_set_images_dir_fd(cur_imgdir);
+}
+
+#define SUCC_ECODE 42
```

Comment:
It looks like this constant definition is duplicated in `test_notify.c`, `test_sub.c`, `test_pre_dump.c` and `test_iters.c`.

Would it be possible to move it in `lib.h`?

Reply:
done